package/boot/uboot-sunxi/Makefile

```diff
@@ -357,6 +366,7 @@ UBOOT_TARGETS := \
 	nanopi_neo_plus2 \
 	nanopi_neo2 \
 	nanopi_r1 \
+        nanopi_r1s_h5 \
```

Comment:
Please use tabs for indention here

Reply:
is it ok now ?

Reply:
No this is still using spaces instated of tabs.